ci/run-wasi-nn-example.sh

```diff
@@ -19,13 +19,37 @@ else
     REMOVE_TMP_DIR=0
 fi
 
+# Set up the environment for OpenVINO on OSX. This is necessary because on OSX, DYLD_LIBRARY_PATH will be ignored when
+# running this file as a script. See: https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_cs_allow-dyld-environment-variables?language=objc
+#
+# For example, if OpenVINO is installed in `~/`, then you would execute this test like the following:
+# OSX_OPENVINO_LIB_PATH="/Users/${USER}/openvino_2022.3.1/runtime/lib/arm64/Release" ./ci/run-wasi-nn-example.sh
+#
+# Note: I found that installing OpenVINO with Homebrew wasn't effective as the directory structure doesn't match the
+# implementation of the library searching in the openvino-rs crates (https://docs.rs/openvino-finder/latest/openvino_finder/).
+# $ tree $(brew --prefix openvino) -d -L 5
+# /opt/homebrew/opt/openvino
+# ├── bin
+# ├── ...
+# └── lib
+#    ├── cmake
+#    │   ├── inferenceengine2023.2.0
+#    │   ├── ngraph2023.2.0
+#    │   └── openvino2023.2.0
+#    ├── openvino-2023.2.0
+#    ├── pkgconfig
+#    └── python3.11
+#        └── ...
+export DYLD_LIBRARY_PATH="${OSX_OPENVINO_LIB_PATH}"
```

Comment:
I opened https://github.com/abrown/install-openvino-action/issues/29 to track this.